templates/docs/examples/patterns/search-and-filter/chip-overflow.html

```diff
@@ -0,0 +1,128 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Search and filter / With chips{% endblock %}
```

Comment:
Duplicated title. Should it be called 'Chips overflow' or something like that?
```suggestion
{% block title %}Search and filter / Chips overflow{% endblock %}
```